packages/saslprep/src/generate-code-points.ts

```diff
@@ -47,8 +41,7 @@ memory.push(
   traverse(bidirectional_l, codePoints.bidirectional_l)
 );
 
-const fsStream = createWriteStream(process.argv[2]);
-fsStream.write(
+process.stdout.write(
```

Comment:
This would also need to keep using `createWriteStream` (or otherwise the package.json bits also need to come back, but I think it might be weird to create one file by writing to stdout and one by writing to disk)